File: code_generator.js
// Word banks for code combinations
const firstWordBank = [
    'PEANUT',
    'EXIT',
    'TUNE',
    'BONFIRE',
    'JAW',
];

const secondWordBank = [
    'ASTRONAUT',
    'MONSTERS',
    'SOUR',
    'BLEND',
    'PLANE',
    'PEANUT',
];

const thirdWordBank = [
    'COCONUT',
    'SUIT',
    'BLUE',
    'ROCKY',
    'MOON',
    'TREETOP',
];

const fourthWordBank = [
    'FIELD',
    'PLANE',
    'CLOUD',
    'NUKE',
    'SHOWER',
    'BUGS',
    'PEAK',
];

// Function to permeate every possible combination
function getCodeOptions() {
    const options = [];

    for (const firstWord of firstWordBank) {
        for (const secondWord of secondWordBank) {
            for (const thirdWord of thirdWordBank) {
                for (const fourthWord of fourthWordBank) {
                    options.push(`${firstWord} ${secondWord} ${thirdWord} ${fourthWord}`);
                }
            }
        }
    }

    return options;
};

// Get options and shuffle the list
const codeOptions = getCodeOptions();
const shuffledOptions = codeOptions.sort(() => Math.random() - 0.5);

// Log each code option line-by-line
shuffledOptions.forEach((code) => {
    console.log(code);
});

File: code_generator.rb
def get_code_options
  first_word_bank = [
    'PEANUT',
    'EXIT',
    'TUNE',
    'BONFIRE',
    'JAW',
    'MOTHER',
  ]

  second_word_bank = [
    'ASTRONAUT',
    'MONSTERS',
    'SOUR',
    'BLEND',
    'PLANE',
  ]

  third_word_bank = [
    'COCONUT',
    'SUIT',
    'BLUE',
    'ROCKY',
    'MOON',
    'TREETOP',
  ]

  fourth_word_bank = [
    'FIELD',
    'PLANE',
    'CLOUD',
    'NUKE',
    'SHOWER',
    'BUGS',
  ]

  options = []

  first_word_bank.each do |first_word|
    second_word_bank.each do |second_word|
      third_word_bank.each do |third_word|
        fourth_word_bank.each do |fourth_word|
          options << "#{first_word} #{second_word} #{third_word} #{fourth_word}"
        end
      end
    end
  end
  return options
end

code_options = get_code_options()
shuffled_options = code_options.shuffle

shuffled_options.each do |code|
  puts code
end
